learn-array-methods-by-building-a-statistics-calculator/styles.css

```diff
@@ -0,0 +1,18 @@
+body {
+  margin: 0;
+  background-color: rgb(27, 27, 50);
+  text-align: center;
+  color: #fff;
+}
+
+button {
+  background-color: rgb(59, 59, 79);
+  border: 3px solid white;
+  color: white;
```

Comment:
I think it would be nice to add the `cursor:pointer;` here

Reply:
I was thinking this too, but when I did some digging, it seems like using `cursor: pointer` is only used for links by default.

It might be better to leave the cursor the way it is, and to use a hover effect over the button since it's styled.

But I have no strong feelings either way. I'm pretty sure that some of the other projects I worked on have a modified cursor for buttons.

I think as long as we're pretty consistent throughout the projects that either way is fine.

Reply:
I was thinking this too, but when I did some digging, it seems like using `cursor: pointer` is only used for links by default.

It might be better to leave the cursor the way it is, and to use a hover effect over the button since it's styled.

But I have no strong feelings either way. I'm pretty sure that some of the other projects I worked on have a modified cursor for buttons.

I think as long as we're pretty consistent throughout the projects that either way is fine.

Reply:
The mouse hover over non-links issues comes up quite a bit in accessibility. By default the browser only uses the pointer for links. I believe this mimics native desktop apps. And then you would use some other styling to indicate the hover state for buttons and such. Some people think this is the way it should always be and others argue that web applications aren't native applications and there can be different rules. 

Personally, I don't consider it a sin to change the mouse to pointer on hover over buttons and such, but I don't usually do that unless it is specifically ask for. So I wouldn't go out of the way to make a point that you need to change the mouse cursor on hover over non-links, because you don't. Leaving the browser defaults is perfectly fine. But if changing the mouse cursor adds something to this then sure, leave it in. I doubt you would ever be docked points in an accessibility audit for it.

Reply:
Awesome, thanks for confirming that @bbsmooth. I wasn't able to find a definitive answer either way, but it's great to know that changing the pointer type for buttons isn't a bad thing accessibility wise.

I believe the other JS projects all use pointers for buttons, so we may stick with that to keep things consistent. If we get enough feedback later saying that we should stick to browser defaults we can do that down the road.